src/datepicker/datepicker.component.ts

```diff
@@ -184,6 +193,8 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 	constructor(protected elementRef: ElementRef) { }
 
 	ngOnChanges(changes: SimpleChanges) {
+		// Global localization of flatpickr instances
+		flatpickr.localize(languages.default[this.language]);
```

Comment:
This changes the config globally ... which is probably what we want in most cases, but slightly inconsistent with how the options are presented (and doesn't allow for per-calendar language overrides) ... this should be something handled by the `flatpickrOptions` getter

Global translations are handled by the `i18n` service ... that's out of scope for this PR though, we can look at that later

Reply:
Changing the language config globally should be left as an option to be done in the DatePicker module imho as in my PR here https://github.com/IBM/carbon-components-angular/pull/976/files#diff-1e9013bc3a56c5ec0f998d5ec536e11cL19.